src/memory_manager.cpp

```diff
@@ -20,8 +20,17 @@
 #include <rmm/mr/device/cuda_memory_resource.hpp>
 #include <rmm/mr/device/default_memory_resource.hpp>
 #include <rmm/mr/device/managed_memory_resource.hpp>
+#include <rmm/mr/device/logging_resource_adaptor.hpp>
 
 namespace rmm {
+
+using cuda_mr = rmm::mr::cuda_memory_resource;
+using pool_mr = rmm::mr::cnmem_memory_resource;
+using managed_mr = rmm::mr::cnmem_managed_memory_resource;
```

Comment:
Was this change intentional, or should we still be using using `rmm::mr::managed_memory_resource` ?